src/LibraryManager.Contracts/FileHelpers.cs

```diff
@@ -210,8 +210,9 @@ public static bool AreFilesUpToDate(FileInfo firstFile, FileInfo secondFile)
         /// Deletes files 
         /// </summary>
         /// <param name="filePaths"></param>
+        /// <param name="rootDirectory"></param>
         /// <returns></returns>
-        public static bool DeleteFiles(IEnumerable<string> filePaths)
+        public static bool DeleteFiles(IEnumerable<string> filePaths, string rootDirectory = null)
         {
             HashSet<string> directories = new HashSet<string>();
```

Comment:
You need to pass in a comparer for this hashset

Reply:
Adding the comparer for the local path, need to further support for network path later.

---
In reply to: [200727108](https://github.com/aspnet/LibraryManager/pull/286#discussion_r200727108) [](ancestors = 200727108)